### PR TITLE
workaround for the OpenBLAS v0.3.1 matrices multiplication issue

### PR DESCRIPTION
**Pull request by pescobar**
kudos to @cfenoy for figuring out this

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.1-GCC-7.3.0-2.30.eb [hunk 1]
```diff
@@ -9,22 +9,30 @@ toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
 # cfr. https://github.com/easybuilders/easybuild-easyconfigs/issues/7180
 toolchainopts = {'vectorize': False}
 
-source_urls = [
+source_urls = [
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.1-GCC-7.3.0-2.30.eb]: @pescobar Please strip off the trailing whitespace (also on other lines below)

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.1-GCC-7.3.0-2.30.eb [hunk 2]
```diff
@@ -9,22 +9,30 @@ toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
...
 ]
-checksums = [
+checksums = [ 
     '1f5e956f35f3acdd3c74516e955d797a320c2e0135e31d838cbdb3ea94d0eb33',  # v0.3.1.tar.gz
     'f328d88b7fa97722f271d7d0cfea1c220e0f8e5ed5ff01d8ef1eb51d6f4243a1',  # large.tgz
     '999c65f8ea8bd4eac7f1c7f3463d4946917afd20a997807300fe35d70122f3af',  # timing.tgz
+    'c85de436d6fff5d9cec1e24127ea9921551cdee373319dbade922d5cd3facd6a',
 ]
 
+buildopts = 'USE_SIMPLE_THREADED_LEVEL3=1'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.1-GCC-7.3.0-2.30.eb]: @pescobar Please add a comment above this line to explain why this is being added?

Comment (pescobar) [reply easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.1-GCC-7.3.0-2.30.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2022.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0672700f3673849e98367bf6a1bb0e4f for a full test report.

**Comment by boegel** [comment]
I was able to reproduce the problem on both Intel Sandy Bridge and Intel Haswell using the scripts from https://github.com/eylenth/Openblas_matrix_issue, and the problem was indeed resolved after rebuilding `OpenBLAS` 0.3.1 using this patch.

Thanks a lot @cfenoy!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3121.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/bd2d323828c3f4593030a0057e9feb4b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c07862385b442b8121160b1535a994a5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**